mods/stairs/init.lua

```diff
@@ -69,10 +69,10 @@ local function get_node_vars(nodename)
 	local def = minetest.registered_nodes[nodename]
```

Comment:
If I'm seeing this correctly, the only thing this function is ever used for is to "transplant" a few definition fields? Why not refactor it to be a `local function transplant_def_fields(nodename, to_def)`?

The code would then become something like

```lua
local def = minetest.registered_nodes[nodename]
if not def then return end
for _, field in ipairs({"light_source", "use_texture_alpha", "sunlight_propagates", "sounds"}) do
    if to_def[field] == nil then
        to_def[field] = def[field]
    end
end
```

Or alternatively, something like `register_node_based_on(nodename, base_nodename, def)`?